lib/MusicBrainz/Server/Data/EntityAnnotation.pm

```diff
@@ -118,18 +119,24 @@ sub merge
         )
     };
 
+    my $current_target_annotation =
+        $self->c->model($model)->annotation->get_latest($new_id);
+    my $current_target_annotation_text = $current_target_annotation
+        ? $current_target_annotation->text
+        : '';
```

Comment:
Is there a reason this wouldn't also work?
```suggestion
    my $current_target_annotation_text = $entity_to_annotation{$new_id} // '';
```

Reply:
... I totally didn't see that option, but that seems perfect and so obvious in hindsight :eyes: Thanks.